test/functional/api/v2/hostgroups_controller_test.rb

```diff
@@ -38,6 +40,13 @@ class HostgroupsControllerTest < ActionController::TestCase
         response = JSON.parse(@response.body)
         assert_job_invocation_is_ok(response, target.map(&:id))
       end
+
+      test 'should list ansible roles for a host' do
+        @host3.hostgroup.ansible_roles = [@ansible_role1]
+        get :ansible_roles, :params => { :id => @host3.hostgroup.id }
+        response = JSON.parse(@response.body)
+        refute_empty response
```

Comment:
An assertion that the response includes the roles name would be better.

`refute_empty` would also pass when the response is an empty JSON Object/Array.

Reply:
> `refute_empty` would also pass when the response is an empty JSON Object/Array.
That's not right, but it would be good to confirm that the right role is returned.